.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7.3.3
+  hmpps: ministryofjustice/hmpps@7
```

Comment:
think this is 8 now

Reply:
or 9 actually https://github.com/ministryofjustice/hmpps-circleci-orb?tab=readme-ov-file